Missionframework/scripts/server/civrep/ignored/chernarus2020.sqf

```diff
@@ -0,0 +1,159 @@
+	"Land_Ind_Timbers",
+	"Land_IndPipe1_stair",
+	"Land_IndPipe2_big_18",
+	"Land_IndPipe2_big_18ladder",
+	"Land_IndPipe2_big_9",
+	"Land_IndPipe2_big_ground1",
+	"Land_IndPipe2_big_ground2",
+	"Land_IndPipe2_big_support",
+	"Land_IndPipe2_bigBuild1_R",
+	"Land_IndPipe2_bigBuild2_L",
+	"Land_IndPipe2_bigL_L",
+	"Land_IndPipe2_bigL_R",
+	"Land_IndPipe2_Small_9",
+	"Land_IndPipe2_Small_ground1",
+	"Land_IndPipe2_Small_ground2",
+	"Land_IndPipe2_SmallBuild1_R",
+	"Land_IndPipe2_SmallBuild2_R",
+	"Land_IndPipe2_SmallL_L",
+	"Land_IndPipe2_SmallL_R",
+	"Land_IndPipe2_T_L",
+	"Land_IndPipe2_T_R",
+	"Land_KBud",
+	"Land_Kontejner",
+	"Land_Lampa_ind",
+	"Land_Lampa_ind_zebr",
+	"Land_Lampa_sidl",
+	"Land_Lampa_sidl_2",
+	"Land_Lampa_sidl_3",
+	"Land_loco_742_blue",
+	"Land_Misc_Cargo1Ao",
+	"Land_Misc_Cargo1B",
+	"Land_Misc_Cargo1Bo",
+	"Land_Misc_Cargo1C",
+	"Land_Misc_Cargo1D",
+	"Land_Misc_Cargo2B",
+	"Land_Misc_Cargo2C",
+	"Land_Misc_Cargo2D",
+	"Land_Misc_Cargo2E",
+	"Land_Misc_deerstand",
+	"Land_Misc_GContainer_Big",
+	"Land_Misc_Scaffolding",
+	"Land_Molo_drevo_bs",
+	"Land_Molo_drevo_end",
+	"Land_Nasypka",
+	"Land_Nav_Boathouse_PierT",
+	"land_nav_pier_c",
+	"land_nav_pier_c_270",
+	"land_nav_pier_c_90",
+	"land_nav_pier_c_big",
+	"land_nav_pier_C_L10",
+	"land_nav_pier_C_R10",
+	"land_nav_pier_C_R30",
+	"land_nav_pier_c_t15",
+	"land_nav_pier_c_t20",
+	"land_nav_pier_c2",
+	"land_nav_pier_c2_end",
+	"land_nav_pier_F_17",
+	"land_nav_pier_F_23",
+	"Land_nav_pier_m_2",
+	"land_nav_pier_m_end",
+	"Land_Plot_green_branka",
+	"Land_Plot_green_vrata",
+	"Land_Plot_rust_branka",
+	"Land_Plot_rust_vrata",
+	"Land_PowLines_ConcL",
+	"Land_PowLines_WoodL",
+	"Land_Psi_bouda",
+	"Land_Pumpa",
+	"Land_Rail_Semafor",
+	"Land_Rail_Zavora",
+	"Land_ruin_wall",
+	"Land_seno_balik",
+	"Land_Shed_M02",
+	"Land_sign_altar",
+	"Land_sign_balota",
+	"Land_Sign_Bar_RU",
+	"Land_sign_berezino",
+	"Land_Sign_BES",
+	"Land_sign_chernogorsk",
+	"Land_sign_elektrozavodsk",
+	"Land_sign_kamenka",
+	"Land_sign_kamyshovo",
+	"Land_sign_komarovo",
+	"Land_sign_solnichnyi",
+	"Land_SignB_Gov",
+	"Land_SignB_GovPolice",
+	"Land_SignB_GovSchool",
+	"Land_SignB_GovSign",
+	"Land_SignB_Hotel_CZ2",
+	"Land_SignB_PostOffice",
+	"Land_SignB_Pub_CZ1",
+	"Land_SignB_Pub_CZ2",
+	"Land_SignB_Pub_CZ3",
+	"Land_SignB_Pub_RU1",
+	"Land_SignB_Pub_RU2",
+	"Land_SignB_Pub_RU3",
+	"Land_sloup_vn",
+	"Land_sloup_vn_drat",
+	"Land_sloup_vn_drat_d",
+	"Land_sloup_vn_dratZ",
+	"Land_Stoplight02",
+	"Land_Tec",
+	"Land_Trafostanica_velka",
+	"Land_Trafostanica_velka_draty",
+	"Land_Vez",
+	"Land_Vez_Pec",
+	"Land_wagon_box",
+	"Land_Wall_CBrk_5_D",
+	"Land_Wall_CGry_5_D",
+	"Land_Wall_Gate_Ind1_L",
+	"Land_Wall_Gate_Ind1_R",
+	"Land_Wall_Gate_Ind2A_L",
+	"Land_Wall_Gate_Ind2A_R",
+	"Land_Wall_Gate_Ind2B_L",
+	"Land_Wall_Gate_Ind2B_R",
+	"Land_Wall_Gate_Ind2Rail_L",
+	"Land_Wall_Gate_Ind2Rail_R",
+	"Land_Wall_Gate_Kolchoz",
+	"Land_Wall_Gate_Village",
+	"Land_Wall_Gate_Wood1"
+];
```

Comment:
missing endline at end of the file, also we're using spaces in new files, it's recommended to use editor with .editorconfig support.

Reply:
RIP, let me go fix that rq